service/controller/rbac/resource/fluxauth/create.go

```diff
@@ -102,6 +102,28 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		return microerror.Mask(err)
 	}
 
+	// create a RoleBinding granting :
+	// - write-silences access for "automation" ServiceAccount *in this org namespace*
+	clusterRoleBinding := &rbacv1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: pkgkey.WriteSilencesAutomationSARoleBindingName(),
+			Labels: map[string]string{
+				label.ManagedBy: project.Name(),
+			},
+			Namespace: ns.Name,
```

Comment:
The `namespace` here will probably not do anything since it's cluster scoped. Did you mean to use a roleBinding?

Reply:
I use namespace to refer to orgs SA's to assign to the ClusterRoleBinding, done in `createOrUpdateClusterRoleBinding`

Reply:
Ah okay, so the namespace is the namespace of the service account that is added to the subjects. Makes sense.
I think still it is quite confusing that the namespace is set here and also the logs appearing that say `creating clusterrolebinding %#q in namespace %s` will make me think that the clusterrolebinding is in a specific namespace when it is not.
If this works I'm okay with it functionally bit I'd prefer if we could remove the namespace part here to ease confusion. If you want to keep visibility for the namespace maybe nstead you could add a logging message like `adding serviceaccount x in namespace y to clusterrolebinding`.

Reply:
removed the namespace as it is not needed here indeed, renamed the logging messages :)